client/app/iplb/frontends/iplb-frontends-edit.controller.js

```diff
@@ -97,6 +97,7 @@ class IpLoadBalancerFrontendsEditCtrl {
                 this.type = "http";
                 this.frontend.port = 443;
                 this.frontend.ssl = true;
+                this.frontend.hsts = false;
```

Comment:
why is it in that pull request??

Reply:
nvm

Reply:
Indeterminate state on checkboxes has breaking changes too.